src/pages/home/report/ReportActionItem.js

```diff
@@ -149,6 +149,32 @@ function ReportActionItem(props) {
         setIsContextMenuActive(isActiveReportActionForMenu);
     }, [isActiveReportActionForMenu]);
 
+    useEffect(() => {
+        const {action, report} = props;
+        const {actionName, errors} = action;
+        const {stateNum, statusNum} = report;
+
+        // We should clear error when two users simultaneously click Mark as Done, Re-open or Cancel button
+        const clearErrorsConditions = {
+            /**  Mark as Done */
+            [CONST.REPORT.ACTIONS.TYPE.TASKCOMPLETED]: stateNum === CONST.REPORT.STATE_NUM.SUBMITTED && statusNum === CONST.REPORT.STATUS.APPROVED,
+            /**  Re-open */
+            [CONST.REPORT.ACTIONS.TYPE.TASKREOPENED]: stateNum === CONST.REPORT.STATE_NUM.OPEN && statusNum === CONST.REPORT.STATUS.OPEN,
+            /** Cancel */
+            [CONST.REPORT.ACTIONS.TYPE.TASKCANCELLED]: stateNum === CONST.REPORT.STATE_NUM.SUBMITTED && statusNum === CONST.REPORT.STATUS.CLOSED,
+        };
+
+        const shouldClearError = (actionType) => actionName === actionType && !_.isEmpty(errors) && clearErrorsConditions[actionType];
+
+        if (
+            shouldClearError(CONST.REPORT.ACTIONS.TYPE.TASKREOPENED) ||
+            shouldClearError(CONST.REPORT.ACTIONS.TYPE.TASKCOMPLETED) ||
+            shouldClearError(CONST.REPORT.ACTIONS.TYPE.TASKCANCELLED)
+        ) {
+            ReportActions.clearReportActionErrors(report.reportID, action);
+        }
+    }, [props, props.action, props.report]);
```

Comment:
@dukenv0307 Do we need to include props in the dependency list?

Reply:
Can you please recheck at least in a platform to ensure that the removal has no affect on the solution.